lines/client/rundown.py

```diff
@@ -1,57 +1,89 @@
 """Client for TheRundown API."""
 import datetime
+import logging
 import os
 import requests
 
+from prefect import Task
+
 from lines.client.config import RundownSportId
-from lines.libs import LinesResponseFacade
+from lines.libs.errors import InvalidSportIdError
+from lines.libs.facades import LinesResponseFacade
+
+logging.basicConfig(level="INFO")
+_LOGGER = logging.getLogger(__name__)
```

Comment:
For now, let's separate to a `logger.py`. I will replace with `betfund_logging` package.